DXRModules/DeusEx/Classes/DXRCameraModes.uc

```diff
@@ -58,22 +58,47 @@ function bool IsFixedCamGame()
 
 ///////////////////////////////////////////////////////////////////
 
-function EnableTempThirdPerson()
+function EnableTempThirdPerson(optional bool noLaser)
 {
+    local #var(PlayerPawn) player;
+
     tempCameraMode=CM_ThirdPerson;
     SetCameraMode(tempCameraMode);
+
+    if (noLaser) {
+        player = class'DXRando'.default.dxr.player;
```

Comment:
you can just do `player()`, see examples below in the old code, like in `function SetFirstPersonCamera()`

Reply:
Is there any advantage there?

Reply:
just cleanliness, Law of Demeter